config/jobs/kubernetes/kops/kops-presubmits.yaml

```diff
@@ -1,6 +1,7 @@
 presubmits:
   kubernetes/kops:
   - name: pull-kops-build
+    cluster: eks-prow-build-cluster
```

Comment:
I thought this job uploaded, in which case credentials could be tricky, but actually it doesn't appear to do so 👍